Lib/re/_constants.py

```diff
@@ -17,10 +17,10 @@
 
 from _sre import MAXREPEAT, MAXGROUPS
 
-# SRE standard exception (access as sre.error)
+# SRE standard exception (access as sre.ReCompileError)
```

Comment:
```suggestion
# SRE standard exception (access as sre.error)
```
I think this should be left alone for this issue.  There is no longer an sre module (and SRE was always wrong I think).  I believe all 'sre', as opposed to '_sre', references should be change or deleted as appropriate in another issue.